docs/concepts/cluster-administration/controller-metrics.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Controller manager metrics
+---
+
+{% capture overview %}
+Controller manager metrics provide important insight into performance and health of
+controller manager.
+
+{% endcapture %}
+
+{% capture body %}
+## What are controller metrics
+
+Controller manager metrics provide important insight into performance and health of controller manager.
+These metrics include common Go language runtime metrics such as go_routine count and controller specif
+ic metrics such as
+etcd request latencies or cloudprovider (AWS, GCE, Openstack) api latencies that can be used
+to gauge health of cluster.
+
+Starting from Kubernetes 1.7, detailed Cloudprovider metrics are available for storage operations for GCE, AWS, Vsphere and Openstack.
+These metrics can be used to monitor health of persistent volume operations.
```

Comment:
Only for GCE metrics are available for all operations, not for other cloudproviders..